### PR TITLE
Add 21 descriptions (6 of 6)

### PR DESCRIPTION
**Pull request by rasa**

### CODE REVIEW

File: bucket/thrift.json [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {
     "homepage": "https://thrift.apache.org/",
+    "description": "A software framework, for scalable cross-language services development, which combines a software stack with a code generation engine to build services that work efficiently and seamlessly between C++, Java, Python, PHP, Ruby, Erlang, Perl, Haskell, C#, Cocoa, JavaScript, Node.js, Smalltalk, OCaml and Delphi and other languages.",
```
Comment (Ash258) [inline bucket/thrift.json]:
```suggestion
    "description": "A software framework, for scalable cross-language services development.",
```

File: bucket/vbindiff.json [hunk 2]
```diff
@@ -1,5 +1,6 @@
 {
     "homepage": "https://www.cjmweb.net/vbindiff/",
+    "description": "VBinDiff (Visual Binary Diff) displays files in hexadecimal and ASCII (or EBCDIC).",
```
Comment (Ash258) [inline bucket/vbindiff.json]:
```suggestion
    "description": "Visual Binary Diff displays files in hexadecimal and ASCII (or EBCDIC).",
```

File: bucket/win32-openssh.json [hunk 3]
```diff
@@ -1,5 +1,6 @@
 {
     "homepage": "https://github.com/PowerShell/Win32-OpenSSH",
+    "description": "The premier connectivity tool for remote login with the SSH protocol. (Microsoft port)",
```
Comment (Ash258) [inline bucket/win32-openssh.json]:
```suggestion
    "description": "A suite of secure networking utilities based on the Secure Shell protocol. (Microsoft port)",
```

File: bucket/wixtoolset.json [hunk 4]
```diff
@@ -1,5 +1,6 @@
 {
     "homepage": "http://wixtoolset.org/",
+    "description": "The most powerful set of tools available to create your windows installation experience.",
```
Comment (Ash258) [inline bucket/wixtoolset.json]:
```suggestion
    "description": "Set of tools for windows installer creation.",
```

**Comment by rasa** [comment]
Sorry, all these should have been squash commits. I thought it used to default to that, but it doesn't appear to now. I will be more careful in the future.

**Pull request merged**